### ISSUE TITLE
Import Rx from @cycle/core

### ISSUE THREAD

**Issue by basham** [TylorS/cycle-director#4]
`cycle-director` should make `@cycle/core` a peer dependency and import its Rx library. Because `cycle-director` directly depends on Rx, Rx is duplicated when building the code.

Instead of:

```js
// ./src/index.js
import Rx from 'rx';
```

```json
  "dependencies": {
    "director": "^1.2.8",
    "rx": "^3.0.0"
  },
```

Do:

```js
// ./src/index.js
import {Rx} from '@cycle/core';
```

```json
  "dependencies": {
    "director": "^1.2.8"
  },
  "peerDependencies": {
    "@cycle/core": "3.x.x"
  },
```

**Comment by TylorS** [comment]
I wouldn't be opposed to depending on `@cycle/core` as opposed to Rx
directly. However peer dependences are a deprecated feature to be removed
in npm v3

**Comment by basham** [comment]
Peer dependencies aren't being depreciated. Having them automatically installed is being depreciated.

> We will also be changing the behavior of peerDependencies in npm@3. We won’t be automatically downloading the peer dependency anymore. Instead, we’ll warn you if the peer dependency isn’t already installed. This requires you to resolve peerDependency conflicts yourself, manually, but in the long run this should make it less likely that you’ll end up in a tricky spot with your packages’ dependencies.

http://blog.npmjs.org/post/110924823920/npm-weekly-5

**Comment by TylorS** [comment]
I stand corrected, I guess I was misinformed. I'll make the changes when I
get home. Thanks for the suggestion and the information.

**Comment by basham** [comment]
:+1:

**Comment by TylorS** [comment]
This has been resolved with this [commit](4d0ad52b92aac84e8b4848eb13922af19f19cf8a).
